tccutil.py

```diff
@@ -75,6 +50,42 @@ def sudo_required():
 
 def open_database():
   #------------------------
+parser = argparse.ArgumentParser(description='Modify Accesibility Preferences')
```

Comment:
Sorry about that. Tried to do some fancy hunk based staging stuff when I had another refactor in progress.